apps/pr-deploy-site/pr-deploy-site.js

```diff
@@ -1,6 +1,7 @@
 // If you are adding a new tile into this site, place make sure it is also being copied from `just.config.ts`
 
-const siteInfo = [
+// Syntax here MUST BE IE11-COMPATIBLE
```

Comment:
Mind adding a note as to why we require ie11 compat? (Notably so that we know when we can stop being ie11 compatible.)

Reply:
Added. We could also address this by running babel or something on the files in this package before uploading, but I'd prefer to do any changes like that separately.